[Term Entry]Dart Queue .contains() (#4470)
Codecademy/docs: content/dart/concepts/queue/terms/contains/contains.md

---
Title: '.contains()'
Description: 'Checks if a specified element is present in the queue.'
Subjects:
 -'Code foundations'
 -'Computer science'
Tags:
 -'Dart'
 -'Concepts'
 -'Queue'
CatalogContent:
 -'learn-dart'
 -'paths/computer-science'
---

> The Dart **.contains()** method in the 'Queue' class is used to check if a specified element is present in the queue. 

Reviewer: ```suggestion
The **`.contains()`** method in Dart checks if a specified element is present in the given queue. 
```

> ## Sintax:

Reviewer: ```suggestion
## Syntax
```
Typo here.

> The syntax for this method is as follows:

Reviewer: ```suggestion
```
We don't need this line so better to remove it.

```pseudo
bool contains(Object element)
```

> - `element`: The element is the object that needs to be checked in the queue.

Reviewer: ```suggestion

- `element`: It is the object that needs to be checked in the queue.
```
Give one line space before and after a heading/codeblock.

> ## Example-1:

Reviewer: ```suggestion
## Example 1
```

```dart

> var queue = Queue();
> queue.addAll([1, 2, 3, 4, 5]);
> print(queue.contains(3)); // Output will be true

Reviewer: ```suggestion
import 'dart:collection';

void main() {
  var queue = Queue();
  queue.addAll([1, 2, 3, 4, 5]);
  print(queue.contains(3)); 
}
```
You need to import the collection library and wrap the code inside the main function.

```

> This method returns `true` if the element is found in the queue, otherwise it returns `false`.

Reviewer: ```suggestion

This method returns `true` if the element is found in the queue, otherwise it returns `false`.
```
Add the output block here too.

> ## Example-2:

Reviewer: ```suggestion
## Example 2
```

```dart

> var names = Queue<String>();
> names.addAll(['Alice', 'Bob', 'Charlie', 'David']);
> print(names.contains('Bob')); // Output will be true

Reviewer: ```suggestion
import 'dart:collection';

void main() {
  var names = Queue<String>();
  names.addAll(['Alice', 'Bob', 'Charlie', 'David']);
  print(names.contains('Bob')); 
}
```

```
In this example, we first add a collection of names to the queue and then use the '.contains()' method to check if the name 'Bob' is present in the queue. The method will return `true` in this case because 'Bob' is indeed present in the queue.

> 

Reviewer: Add the output block.

> 

Reviewer: Remove these unnecessary spaces from the content.

---
Title: '.contains()'
Description: 'Checks if a specified element is present in the queue.'
Subjects:
 -'Code foundations'
 -'Computer science'
Tags:
 -'Dart'
 -'Concepts'
 -'Queue'
CatalogContent:
 -'learn-dart'
 -'paths/computer-science'
---

The **`.contains()`** method in Dart checks if a specified element is present in the given queue. 

## Syntax

```pseudo
bool contains(Object element)
```

- `element`: It is the object that needs to be checked in the queue.

## Example 1

In the following example, the code checks whether the `queue` contains an element `3` or not:

```dart
import 'dart:collection';

void main() {
  var queue = Queue();
  queue.addAll([1, 2, 3, 4, 5]);
  print(queue.contains(3)); 
}
```

> The output of the above is:

Reviewer: ```suggestion
The output of the above code is:
```

---

> Title: '.contains()'
> Description: 'Checks if a specified element is present in the queue.'
> Subjects: -'Code foundations'
>   -'Computer science'
> Tags: -'Dart'
>   -'Concepts'
>   -'Queue'
> CatalogContent: -'learn-dart'
>   -'paths/computer-science'

Reviewer: ```suggestion
Title: '.contains()'
Description: 'Checks if a specified element is present in the queue.'
Subjects: 
  - 'Code foundations'
  - 'Computer science'
Tags: 
  - 'Dart'
  - 'Concepts'
  - 'Queue'
CatalogContent: 
  - 'learn-dart'
  - 'paths/computer-science'
```
